### PR TITLE
cli: print conflicted paths whenever the working copy is changed

### PR DESCRIPTION
**Pull request by bnjmnt4n**
This is disabled when the global `--quiet` flag is used, as suggested by https://github.com/martinvonz/jj/pull/975#issuecomment-1371580761.

# Checklist

If applicable:
- [X] I have updated `CHANGELOG.md`
- [ ] I have updated the documentation (README.md, docs/, demos/)
- [ ] I have updated the config schema (cli/src/config-schema.json)
- [ ] I have added tests to cover my changes

### CODE REVIEW

File: cli/src/cli_util.rs
Comment (yuja) [inline cli/src/cli_util.rs:2243]: I expect the global `--quiet` will suppress most status messages ("Nothing changed.", "Rebased", "Working copy now at:", etc.) Is that the plan?

File: cli/src/cli_util.rs
Comment (yuja) [inline cli/src/cli_util.rs:1167]: nit: just leave the message unlabeled? labeling it as `"conflict"` (of `jj log`) seems odd.

File: cli/src/cli_util.rs
Comment (bnjmnt4n) [reply cli/src/cli_util.rs:2243]: I'm not too sure, perhaps this should be discussed. I just added it since `jj resolve` also had the `--quiet` flag for muting the list of conflicts, which I think might be handy if there are a lot of conflicted paths.

Comment (bnjmnt4n) [reply cli/src/cli_util.rs:2243]: I will rebase this after #3401 is merged.

Comment (bnjmnt4n) [reply cli/src/cli_util.rs]: Rebased!

File: cli/src/commands/resolve.rs
Comment (yuja) [inline cli/src/commands/resolve.rs:115]: nit: `workspace_command.get_wc_commit_id()`

**Review by yuja — APPROVED** [review]
lgtm, thanks.

**Comment by bnjmnt4n** [comment]
Do we have a process on number of contributor approvals required before PRs should be merged? Since this change potentially affects the output of all commands, not sure if more contributors should agree with the changes before merging.

**Comment by yuja** [comment]
> Do we have a process on number of contributor approvals required before PRs should be merged?

No, I don't think there is.

> Since this change potentially affects the output of all commands, not sure if more contributors should agree with the changes before merging.

I think this change was discussed on Discord, so I'm not too worried about the change. But just in case, cc @martinvonz

**Comment by martinvonz** [comment]
> Do we have a process on number of contributor approvals required before PRs should be merged?

No, there isn't (except that there has to be at least one, of course), but thanks for asking. Today is release day, though, so it means we won't get any time to test this before the release. I want this feature myself, but maybe it's best to merge it in a few hours after the release.

**Comment by martinvonz** [comment]
Since the release is now done, feel free to merge this

**Review by thoughtpolice — APPROVED** [review]
Seems nice; LGTM.

> Do we have a process on number of contributor approvals required before PRs should be merged?

It's been mentioned in brief a few times in the past but, at some point in the nebulous future, we'll probably do the "at least 2 humans rule", i.e. every change must at least have two approvals. But it's not clear we need to do that yet.

**Pull request merged**
